pkgs/top-level/aliases.nix

```diff
@@ -1695,6 +1695,8 @@ mapAliases {
   xcodebuild6 = throw "'xcodebuild6' has been renamed to/replaced by 'xcodebuild'"; # Added 2023-12-10
   xdg_utils = xdg-utils; # Added 2021-02-01
   xen-light = throw "'xen-light' has been renamed to/replaced by 'xen-slim'"; # Added 2024-06-30
+  xenPackages.xen_4_16 = throw "While Xen 4.16 was still security-supported when it was removed from Nixpkgs, it would have reached its End of Life a couple of days after NixOS 24.11 released. To avoid shipping an insecure version of Xen, the Xen Project Hypervisor Maintenance Team decided to delete the derivation entirely."; # Added 2024-09-29
+  xenPackages.xen_4_17 = throw "Due to technical challenges involving building older versions of Xen with newer dependencies, the Xen Project Hypervisor Maintenance Team decided to switch to a 'n+n-1' support cycle. As Xen 4.17 would have been the 'n-2' version, it was removed."; # Added 2024-09-29
```

Comment:
Hmm, I guess this works as long as `xenPackages` is not meant to exist. But maybe `xenPackages = throw "…use the top‐level packages please…";` is more informative, given people are more likely using the newer versions from this package set if anything anyway?